api/kyverno/v1/spec_types.go

```diff
@@ -13,12 +13,21 @@ type ValidationFailureAction string
 
 // Policy Reporting Modes
 const (
-	// Enforce blocks the request on failure
-	Enforce ValidationFailureAction = "enforce"
-	// Audit indicates not to block the request on failure, but report failures as policy violations
-	Audit ValidationFailureAction = "audit"
+	// enforceOld blocks the request on failure
+	// DEPRECATED: use enforce instead
+	enforceOld ValidationFailureAction = "enforce"
+	// enforce blocks the request on failure
+	enforce ValidationFailureAction = "Enforce"
```

Comment:
Where are "audit" and "auditOld" defined? Or we don't need it anymore?

Reply:
Not needed anymore. Audit is when not in enforce mode so we just need to check enforce mode.